packages/split-button/sync/sp-split-button.ts

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { Picker } from '@spectrum-web-components/picker';
+import {
+    Overlay,
+    OverlayOptions,
+    TriggerInteractions,
+} from '@spectrum-web-components/overlay';
+import '../sp-split-button.js';
+
+Picker.openOverlay = async (
+    target: HTMLElement,
+    interaction: TriggerInteractions,
+    content: HTMLElement,
+    options: OverlayOptions
+): Promise<() => void> => {
+    return await Overlay.open(target, interaction, content, options);
+};
```

Comment:
it might be worth DRYing up the `openOverlay` overrides into a Picker import.

Reply:
Agreed!